include/fmt/format-inl.h

```diff
@@ -819,6 +817,16 @@ struct uint128_wrapper {
   }
 };
 
+// Compilers should be able to optimize this into the ror instruction.
+inline std::uint32_t rotr(uint32_t n, uint32_t r) noexcept {
```

Comment:
nit: I suggest dropping `std::` in the return type for consistency (and the same in the overload below).

Reply:
@vitaut Sorry, I think I was in rush for some reason 😅 corrected now.
@miscco Added, thanks for pointing out.

Reply:
@jk-jeon
``FMT_CONSTEXPR`` - macro for C++14 constexpr.

Reply:
@phprus  oooohhh didn't know that, thanks for letting me know=)